app/lib/sidebar/sidebar.dart

```diff
@@ -12,24 +13,21 @@ class SideBar extends StatelessWidget {
           DrawerHeader(
             child: Text('Hello'),
             decoration: BoxDecoration(
-              gradient: LinearGradient(
-                colors: <Color>[
-                  Colors.green,
-                  Colors.greenAccent,
-                ],
-              ),
+              gradient: GradientColors,
             ),
           ),
-          CustomTile(
+          TemplateListTile(
             icon: Icons.star_border,
             text: 'In Progress',
+            textStyle: Theme.of(context).textTheme.bodyText1,
           ),
           SizedBox(
```

Comment:
Is this box a spacer, can we see ourselves reusing this in the future?

Reply:
Size box is like adding an invisible box, but I think we can also use other styling properties as substitute. I only use sizebox widget as part of the experimentation